### DISCUSSION TITLE
Possibility of a CI/CD script for AWS?

### DISCUSSION THREAD

**Discussion by 3steve3** [jasontaylordev/CleanArchitecture#976] [Ideas]
For users where it may be difficult to switch to Azure for their cloud hosting needs, could there also be scripts for CI/CD using AWS?

**Comment by jasontaylordev** [comment]
I think this is a good idea. I wonder if using Terraform might solve this problem? Also, I'm keen to add support for Azure DevOps along with GitHub Actions. If anyone wants to help with adding support for Terraform or Azure DevOps, let me know. 😀

**Reply by VincentVerweij** [reply]
What do you exactly mean when you say support for Azure DevOps? Just a pipeline, like the thing you currently have GitHub?
Also, would you be converting the current Bicep scripts into Terraform to be able to offer AWS next to Azure in the future, or just Terraform for AWS without Azure?
